packages/as_msgs/msg/wheel_odometry.msg

```diff
@@ -0,0 +1,6 @@
+Header header
+float32 x
```

Comment:
I don't quite understand why there is x, y, heading in wheel odometry. Isn't it just the wheel speed?

Reply:
I was thinking of wheel odometry as a current pose estimate calculated over time from the wheel speeds. Is the convention to have it just be wheel speed and have the pose estimate done somewhere else?

Reply:
Yes, I think they are different things. Now, once you change trajectory_point.msg to pose.msg, you can use pose.msg to publish the pose estimate from wheel speeds (or anything else like SLAM in the future). It means you have the same message type regardless of how you compute the pose. So in the future, if you use another way to compute the pose, you don't have to change your planning/control algorithm at all.

Reply:
That all makes sense and makes the codebase more modular, will have us get that done